### DISCUSSION TITLE
FE list view: Which features do you use?

### DISCUSSION THREAD

**Discussion by oliverklee** [oliverklee/ext-seminars#1380] [Ideas]
As part of a customer project, I will probably complete rewrite the FE list view. For this, I plan to only implement the functionality, displayed and hooks that are actually still used/needed. For this, I need your input. :-)

Please comment:
- Which columns in the FE list view do you use?
- Do you need the sorting functionality (which sits behind the column headings)?
- Which of the FE list view hooks do you use? For which purpose?
- Do you use the pagination?
- Which of the date constraints for the events in the list do you use?
  - past events
  - past + ongoing events
  - ongoing events
  - ongoing + future events
  - future events
  - all events
- Do you use the filter form (i.e., the search widget)? If so, which filter fields are important to you?

**Comment by lisardo2** [comment]
First and foremost: it should become a fluid template. Traditional templating is much to restricted. 

Which columns in the FE list view do you use?
- title
- date
- category
- seminar number 
- teaser text

Which of the FE list view hooks do you use? For which purpose?
=> none. But we need the Hooks for the registration mails. 

Do you need the sorting functionality (which sits behind the column headings)?
=> yes, the visitors use it in order to switch from alphabetical list to date list; or grouping the events in different subjects. 

Do you use the pagination?
=> no, we dont need it. 

Which of the date constraints for the events in the list do you use?
=> all of them 

Do you use the filter form (i.e., the search widget)? If so, which filter fields are important to you?
=> no.

**Comment by t3premium** [comment]
In the list view we use:
- title
- teaser
- date (begin_date, end_date)
- place
- vacancies

**Comment by peerlight** [comment]
## Which columns in the FE list view do you use?

- date, time, city, place, title, subtitle, event_type, cancelled (as a status), categories, image

## Do you need the sorting functionality (which sits behind the column headings)?

- no

## Which of the FE list view hooks do you use? For which purpose?

- none

## Do you use the pagination?

- yes

## Which of the date constraints for the events in the list do you use?

- ongoing + future events

## Do you use the filter form (i.e., the search widget)? If so, which filter fields are important to you?

- yes
- filter by date range, city, event_type, organizer, categories, place, target_grup
- search text input

**Comment by jpmschuler** [comment]
As we just have the most basic use of fe_login, select event (no use of actual dates, prices or anything), add notes (no fields except a free field) and join waiting queue, thus:
Title, free spots available, and the register link

No filter and pagination due to the small amount of concurrent events, however sorting by choice(i.e. field `sorting`) instead of rule is deemed relevant for the editors.

**Comment by oliverklee** [comment]
@jpmschuler Thanks! Could you also please comment which seminar fields your list view displays?

**Reply by jpmschuler** [reply]
Really just title and free spots
![48745F09-9F24-4A44-9066-5012DFC61D82](https://user-images.githubusercontent.com/12411176/177162632-4ac839ac-4e48-48f9-b657-c30306989dcb.png)

**Comment by oliverklee** [comment]
From a customer:
- title
- date + time
- place
- vacancies
- price
- register link

**Comment by oliverklee** [comment]
From another customer:
- title
- date
- "more" link
- place
- event type
- vacancies icon
